--- /etc/lightdm/lightdm.conf	2018-10-29 09:52:00.655371324 +0100
+++ /etc/lightdm/lightdm.conf	2019-05-17 16:59:29.000000000 +0200

@@ -99,7 +99,7 @@
 #xdmcp-manager=
 #xdmcp-port=177
 #xdmcp-key=
-#greeter-session=example-gtk-gnome
+greeter-session=lightdm-deepin-greeter
 #greeter-hide-users=false
 #greeter-allow-guest=true
 #greeter-show-manual-login=false
@@ -117,7 +111,7 @@
 session-wrapper=/etc/lightdm/Xsession
 #greeter-wrapper=
 #guest-wrapper=
-#display-setup-script=
+display-setup-script=/usr/bin/setxkbmap fr
 #display-stopped-script=
 #greeter-setup-script=
 #session-setup-script=
